src/components/Directory/Directory.scss

```diff
@@ -67,27 +67,37 @@
       .item-content {
         margin-bottom: 10px;
 
-        a {
-          // color: $mediumGrey;
-          color: $black;
-          font-size: 1rem;
-          text-decoration: none;
-          font-weight: bold;
-          transition: all 0.2s ease-out;
-
+        .icon-title-tor{
           display: flex;
-          align-items: center;
-
-          @media (max-width: $thirdBreak) {
-            font-size: 0.9rem;
-          }
-
-          &:hover {
-            color: $primaryGreen;
+          a {
+            // color: $mediumGrey;
+            color: $black;
```

Comment:
```suggestion
            color: inherit;
```

Reply:
The link split change was good, but now the titles aren't readable in dark mode anymore.

![grafik](https://user-images.githubusercontent.com/886/192604449-e33810db-645e-4f97-a066-41bf0ffe21be.png)

Inheriting the color for the link fixes this.